frontend/src/pages/project.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ProjectComponent from '../components/Project';
+
+class ProjectPage extends React.Component {

```

Comment:
we prefer functional components over the class component style. makes it easier to reason about sequential logic and hooks.